### PR TITLE
feat: add luaurc schema, resolves: #842

### PR DESCRIPTION
**Pull request by ernisto**
still waiting for me PR on schema store to be accepted
https://github.com/SchemaStore/schemastore/pull/4248

### CODE REVIEW

**Comment by ernisto** [comment]
image
my pr on schemastore has been merged

**Comment by ernisto** [comment]
nvm, the schema is kinda so wrong

**Comment by ernisto** [comment]
issues fixed

File: editors/code/schemas/luaurc.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "typeErrors": { "type": "boolean" },
+        "globals": {
+            "type": "array",
+            "items": { "type": "string", "pattern": "^[a-zA-Z_]\\w*" }
+        },
+        "aliases": {
+            "type": "object",
+            "additionalProperties": false,
+            "patternProperties": {
+                "^(?!\\.{1,2}$)(?!.*\\/)[a-zA-Z0-9\\-\\._]+$": {
```
Comment (JohnnyMorganz) [inline editors/code/schemas/luaurc.json:21]: I don't completely get this regex. What is the purpose of the negative lookahead / lookbehind?

**Review by JohnnyMorganz — COMMENTED** [review]
Thanks, looks good! Just one question.

Also, can you add some documentation for the entries where it makes sense? Maybe even just copy it from what is said on https://github.com/luau-lang/rfcs/blob/master/docs/config-luaurc.md

File: editors/code/schemas/luaurc.json (hunk 1 above)
Comment (ernisto) [reply editors/code/schemas/luaurc.json:21]: to reject the string if is exacly "." or ".."

File: editors/code/schemas/luaurc.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                    "type": "string"
+                }
+            }
+        },
+        "lint": {
+            "title": "Lints to enable",
+            "description": "Points to an object that maps string literals that correspond to the names of linting rules (see https://luau-lang.org/lint), or \"*\" that means \"all rules\", to a boolean (to enable/disable the lint)",
+            "default": "nonstrict",
```
Comment (JohnnyMorganz) [inline editors/code/schemas/luaurc.json:63]:
```suggestion
```

**Review by JohnnyMorganz — APPROVED** [review]
Thanks!

**Pull request merged**